app/src/main/java/com/github/braillesystems/learnbraille/res/GolubinaCourse.kt

```diff
@@ -230,9 +230,14 @@ internal val golubinaIntroLessons by lessons {
                     Они получаются из букв А, Б, Ц, Д, Е добавлением цифрового знака.
                     Например, цифра 3, как и число 3 - это цифровой знак + Ц.
                     Число двадцать четыре - это цифровой знак, затем буквы Б и Д. 
-                    В уроках мы для краткости не будем всякий раз ставить цифровой знак."""
+                    В следующих шагах просмотрите, а затем введите число «двенадцать тысяч
+                    триста сорок пять»."""
         )
-        showAndInputChars("12345")
+        +Show(content.markers.getValue(MarkerType.NumberSign))
+        for (char in "12345") {
+            +Input(content.symbols.getValue(char))
```

Comment:
`Show` should be here

Reply:
many thanks! Fixed it. Will check everything one more time before marking this request as ready